src/dotnet-interactive-vscode/src/vscode/notebookProvider.ts

```diff
@@ -31,14 +32,25 @@ export class DotNetInteractiveNotebookContentProvider implements vscode.Notebook
                 break;
         }
 
-        this.clientMapper.getOrAddClient(uri);
+        let client = this.clientMapper.getOrAddClient(uri);
+        let notebookPath = path.dirname(uri.fsPath);
+        client.changeWorkingDirectory(notebookPath).catch((err) => {
+            let message = `Unable to set notebook working directory to '${notebookPath}'.`;
+            if (err && err.message) {
+                message += '\r\n' + err.message.toString();
```

Comment:
Any reason to use the Windows-specific `\r\n`?

Reply:
muscle memory, I'll make it a simple `\n` since it doesn't matter